Tb/capture

tbarthel-fr opened the pull request.

tbarthel-fr committed:
[Capture] display visual callback for the new added color.

Basically, scroll to the top and animate the new added color.

tbarthel-fr committed:
[Capture] pick color one by one for kids.

tbarthel-fr committed:
[Capture] support 10 inch tablette.

vbarthel-fr commented:
We will need to choose a convention for the base/core activity shared by the flavors. In a previous MR (https://github.com/tvbarthel/CameraColorPicker/pull/25/) the name PaletteCreationBaseActivity was used, in this MR the name CoreColorPickerActivity was introduced.

What do you prefer xxxBaseActivity or CoreXxxActivity ?

Otherwise lgtm :+1:

tbarthel-fr committed:
[Capture] apply consistency for the capture activity.

Remove the flavor static class and move the behavior
inside the flavored activity.

Rename the parent activity using xxxBaseActivity
instead of CorexxxActivity.

tbarthel-fr commented:
@vbarthel-fr consistency applied for name and specific flavour code regarding the capture. Could you review the last commit?

vbarthel-fr commented:
lgtm !! :ghost:
